surfer/viz.py

```diff
@@ -556,7 +556,8 @@ def _toggle_render(self, state, views=None):
                 mlab.draw(figure=_f)
                 with warnings.catch_warnings(record=True):  # traits focalpoint
                     mlab.view(*view[0], figure=_f)
```

Comment:
`mlab.view` has a `roll` argument, could it be passed here instead of making a separate call?

Reply:
Yes seems to work too.